templates/ui-frameworks/lit/link-type/ui/src/{{dna_role_name}}/{{coordinator_zome_manifest.name}}/{{#if (and bidireccional (and to_referenceable (ne from_referenceable.hash_type 'AgentPubKey')))}}{{kebab_case (plural from_referenceable.name)}}-for-{{kebab_case to_referenceable.name}}.ts{{¡if}}.hbs

```diff
@@ -46,11 +47,11 @@ export class {{pascal_case (plural from_referenceable.name)}}For{{pascal_case to
   }
 
   renderList(hashes: Array<{{to_referenceable.hash_type}}>) {
-    if (hashes.length === 0) return html`<span>No {{lower_case (plural from_referenceable.name)}} found for this {{lower_case to_referenceable.name}}</span>`;
-    
+    if (!hashes.length) return html`<article>No {{lower_case (plural from_referenceable.name)}} found for this {{lower_case to_referenceable.name}}</article>`;
```

Comment:
Is this an `<article>` tag on purpose?

Reply:
According to the [MDN docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/article) for an article tag it's used for content that's "intended to be independently distributable or reusable" like for example a full forum post. I'm not sure this would apply here...

Reply:
Yes, this was a misused semantic tag, replaced it with a `div`